RecoJets/JetProducers/python/hltPUIdParams_cfi.py

```diff
@@ -0,0 +1,25 @@
+import FWCore.ParameterSet.Config as cms
+full_74x_wp  = cms.PSet(
```

Comment:
why do we need to split off this PSet to a separate file?
Current implementation doesn't include it anywhere else.
Please move the contents to hltPUIdAlgo_cff